common/app/controllers/EmailSignupController.scala

```diff
@@ -38,6 +38,7 @@ object ListIds {
 
   val theBestOfOpinion = 2313
   val newBestOfOpinion = 3811
+  val controlBestOfOpinion = 3814
```

Comment:
this is also the old email, right? (just trying to get it straight in my head. three lists, two of which go to the old email, one of which goes to the new email)